tests/robot-tests/tests/admin_and_public_2/bau/publish_amend_cancel.robot

```diff
@@ -0,0 +1,459 @@
+    ...    Dates test subject test data guidance content
+    user clicks button    Save guidance
+
+Add ancillary file
+    user clicks link    Ancillary file uploads
+    user waits until h2 is visible    Add file to release
+
+    user enters text into element    label:Title    Test ancillary file 1
+    user enters text into element    label:Summary    Test ancillary file 1 summary
+    user chooses file    label:Upload file    ${FILES_DIR}test-file-1.txt
+    user clicks button    Upload file
+
+    user waits until page contains accordion section    Test ancillary file 1
+    user opens accordion section    Test ancillary file 1    id:file-uploads
+
+    ${section_1}=    user gets accordion section content element    Test ancillary file 1    id:file-uploads
+    user checks summary list contains    Title    Test ancillary file 1    ${section_1}
+    user checks summary list contains    Summary    Test ancillary file 1 summary    ${section_1}
+    user checks summary list contains    File    test-file-1.txt    ${section_1}
+    user checks summary list contains    File size    12 B    ${section_1}
+
+    user checks there are x accordion sections    1    id:file-uploads
+
+Navigate to 'Footnotes' section
+    user clicks link    Footnotes
+    user waits until h2 is visible    Footnotes
+
+Add a footnote
+    user clicks link    Create footnote
+    user waits until h2 is visible    Create footnote
+    user clicks footnote subject radio    Dates test subject    Applies to all data
+    user clicks element    id:footnoteForm-content
+    user enters text into element    id:footnoteForm-content
+    ...    Applies to all data 1
+    user clicks button    Save footnote
+    user waits until h2 is visible    Footnotes
+
+Add a second footnote
+    user clicks link    Create footnote
+    user waits until h2 is visible    Create footnote
+    user clicks footnote subject radio    Dates test subject    Applies to all data
+    user clicks element    id:footnoteForm-content
+    user enters text into element    id:footnoteForm-content
+    ...    Applies to all data 2
+    user clicks button    Save footnote
+    user waits until h2 is visible    Footnotes
+
+Create data block table
+    user creates data block for dates csv    Dates test subject    ${DATABLOCK_NAME}    Dates table title
+
+Create chart for data block
+    user waits until page contains link    Chart
+    user waits until page does not contain loading spinner
+    user clicks link    Chart
+
+    user clicks button    Choose an infographic as alternative
+    user chooses file    id:chartConfigurationForm-file    ${FILES_DIR}test-infographic.png
+    user checks radio is checked    Use table title
+    user enters text into element    id:chartConfigurationForm-alt    Sample alt text
+
+    user clicks button    Save chart options
+
+    user waits until page contains    Chart preview
+    user checks infographic chart contains alt    id:chartBuilderPreview    Sample alt text
```

Comment:
Given that we're not changing the chart config as part of creating the amendment, I'd probably cut this part of the test out, as I don't think that it's adding much to the test case. I could be wrong! smile